docs/releases/2.17.md

```diff
@@ -29,3 +29,9 @@ Here are other changes related to the redesign:
 
 ## Upgrade considerations
 
+### Hallo legacy rich text editor has moved to an external package
+
+* Hallo was deprecated in in [Wagtail v2.0 (February 2018)](https://docs.wagtail.org/en/stable/releases/2.0.html#new-rich-text-editor) and has not received any bug fixes with only a minimal level of support since then.
+* If you still require Hallo for your Wagtail installation, you will need to install the [Wagtail legacy Hallo editor](https://github.com/wagtail/wagtail-hallo) package.
+* There will be no ongoing support for this package beyond this release.
```

Comment:
Is it as clear cut as this? In the repo README we've said "Pull requests will be accepted", and it's possible that others in the community will step up to maintain it.

Reply:
Good point. Not sure what to say here. We want to encourage people to take the steps to migrate away from hallo. 

Maybe. 

```
* We encourage all users of the Hallo editor to take steps to migrate to the new Draftail editor as this external package is unlikely to have ongoing maintenance.
```

Reply:
LGTM